### ISSUE TITLE
Feature Request: Add Reblog toggle off option in Reader for Jetpack sites

### ISSUE THREAD

**Issue by StefMattana** [Automattic/wp-calypso#89308]
### Impacted plugin

Social

### What

Ensure the Reblog option on JP site in Reader can be toggled off

### How

This has been reported by a user in 7936169-zen. 

They would like to remove the Reblog option when their Jetpack site appears in the Reader, but this is not possible as there is no Reblog toggle/option in any Jetpack settings > Sharing buttons.

dotcom sites do have the Reblog toggle on/off option instead: https://wordpress.com/support/reblogs/#turn-reblogs-on-or-off

It'd be great to allow the Jetpack sites to toggle off the Reblog feature. 

Noting that the same issue exists with other options as well, for example, the Like button. If the Like button is disabled on the site, it continues to show on Reader.

**Comment by jeherve** [comment]
Moving this to the Calypso repo, where the Reader interface lives.

**Comment by jeherve** [comment]
@Automattic/loop What do you think of this? Should we maybe not display the Reblog button at all for Jetpack sites?

**Comment by davemart-in** [comment]
> Should we maybe not display the Reblog button at all for Jetpack sites?

Tell me more. Curious what the rationale would be for removing it from all JP sites.

**Comment by jeherve** [comment]
The option is not available to display on the self-hosted sites themselves, so it's not something site owners may even know about, or may expect to find under the posts. I could see this as an argument to not display the option in the Reader. I assume this is why this site owner would want an option for their own site.

At the same time, the Reader is its own interface, on its own platform. You could compare it to Facebook ; someone can share a link to a post from your site on their Facebook Profile and their friends will see Reaction options, comments, and a Share option. The "Reblog" option is not so different from that in many ways, I think we could reasonably say that the "Reblog" option can be there just like a "Share" button can be there for a Facebook post, as long as the original content is publicly accessible.

In short, I think there are arguments for both. :)

**Comment by davemart-in** [comment]
@eoigal @Addison-Stavlo @roo2 I know this discussion has come up in the past. Curious to hear your take.

**Comment by eoigal** [comment]
The Reblog (now Repost) option is just another way of sharing some creators post while making sure the original author is credited.

I think we should push back against requests like this. The JP posts are public and can be shared/reposted without the authors consent. The Reblog/Repost option is a passive feature that can benefit the original creator.

FWIW - We can flag the site as `broken-in-reader` (or add support for new flag type) to filter JP creators that wish to not be on Reader - https://opengrok.a8c.com/source/xref/wpcom/wp-content/lib/reader/reader.php?r=c90c6d70#3

**Comment by Addison-Stavlo** [comment]
> I think we should push back against requests like this.

Agreed. "Repost" in the reader only puts that post's URL in an embed block. If your post is public, a URL for it can be shared. Thats really all that is happening and there is no real special functionality having your site in the reader is enabling here. Reader or not, people can take URLs of your posts and drop them in embed blocks. 🤷‍♀️ 

> so it's not something site owners may even know about, or may expect to find under the posts. 

It sounds like another place where we may need to educate or combat mis information better.  This is one reason it was renamed from "reblog" to "repost", because it is not classic reblog functionality - its more sharing or attributing someone else's post to your ideas by including the url to it.
